adapters/repos/db/clusterintegrationtest/helpers_for_test.go

```diff
@@ -158,6 +159,20 @@ func class() *models.Class {
 	}
 }
 
+func multiVectorClass() *models.Class {
+	return &models.Class{
+		Class:               "Test",
+		InvertedIndexConfig: invertedConfig(),
+		VectorConfig: map[string]models.VectorConfig{
+			"custom1": {VectorIndexConfig: enthnsw.UserConfig{}},
+			"custom2": {VectorIndexType: "hnsw", VectorIndexConfig: enthnsw.UserConfig{}},
+			"custom3": {VectorIndexType: "flat", VectorIndexConfig: flatent.UserConfig{}},
+			//"custom4": {VectorIndexType: "dynamic", VectorIndexConfig: dynamicent.UserConfig{}},  // async only
```

Comment:
I think that we can remove this commented part

Reply:
added a test for dynamic indexing